content/automate/ManageIQ/System/Event/EmsEvent/Ansible_Tower.class/_missing.yaml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
@lfu is there a reason to choose `Ansible_Tower.class` rather than `AnsibleTower.class`?